docs/fleet/machines/control.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Control Machines"
+linkTitle: "Control"
+weight: 10
+type: "docs"
+description: "A machine is an organizational concept, consisting of either one or multiple parts working closely together to complete tasks."
+tags: ["fleet management", "control", "app"]
+images: ["/fleet/control.gif"]
+---
+
+Once you have configured components and services for your machine, you can test, monitor, and remotely operate them from the **CONTROL** tab in the [Viam app](https://app.viam.com) or the [Viam mobile app](/fleet/#the-viam-mobile-app).
+
+## Control interface in the Viam app
+
+The **CONTROL** tab in the [Viam app](https://app.viam.com) gives you the ability to test, monitor, and operate the machines in your fleet.
+The **CONTROL** tab provides a control interface for each component and service that you have configured for you machine.
+
+For example, if you have configured a base with wheels, you can move your machine's with an arrow pad and control the base's speed by setting its power with a slider.
+If you have configured a camera component, a window in the **CONTROL** tab displays the camera output.
+
+If you use remote control in the [Viam app](https://app.viam.com) UI, all communication to the machine uses [WebRTC](https://pkg.go.dev/go.viam.com/utils@v0.0.3/rpc#hdr-Connection).
+For local communication between [parts](/build/configure/parts/#machine-parts) Viam uses gRPC or WebRTC.
+
+{{<gif webm_src="/fleet/control.webm" mp4_src="/fleet/control.mp4" alt="Using the control tab" max-width="800px">}}
+
+### Components
+
+For more detailed information on how to operate and test your resources, expand the relevant resource below:
+
+{{% expand "arm" %}}
+{{< readfile "/static/include/components/test-control/arm-control.md" >}}
+{{% /expand%}}
+
+{{% expand "base" %}}
+{{< readfile "/static/include/components/test-control/base-control.md" >}}
```

Comment:
All of these includes files link to `/fleet/machines/#control`, so those should probably be updated to link to `/fleet/machines/control/` instead, but also it's awkward to have them link anywhere in the context of this particular page.

Some of these includes files include a heading and some don't, and the headings are inconsistent (i.e. h2 for arm vs h4 for digital interrupts).

Reply:
Updated link. I think the top of the file stayed fairly similar to the content in as it was in the fleet page, so think that shouldn't be a problem?

Reply:
I mean that it makes sense for the control section of each model to link to this page, but it makes much less sense for sections of this page to link to the page the user is already on. I realize we can't easily change this because we are using the includes files, so maybe leave as-is and hope users don't find it annoying. There's not a way to take the link out conditionally when the includes is included in this page is there?

Reply:
we'd have to essentially make it a shortcode... I'll try

Reply:
because of how the include works it's not straightforward. I think it can be done, I'll make a maintenance ticket for it